svm/api_test.go

```diff
@@ -7,17 +7,41 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func ReadTemplate(t *testing.T, path string) []byte {
+func ReadFile(t *testing.T, path string) []byte {
 	bytes, err := ioutil.ReadFile(path)
 	assert.Nil(t, err)
 	return bytes
 }
 
+func Deploy(t *testing.T, path string) (*Runtime, *DeployReceipt, error) {
+	Init(true, "")
+
+	rt, err := NewRuntime()
+	assert.Nil(t, err);
+
+	msg := ReadFile(t, path)
+	gas := 1000000000
+	env := NewEnvelope(Address{}, Amount(10), TxNonce{Upper: 0, Lower: 0}, Gas(gas), GasFee(0))
+	ctx := NewContext(Layer(0), TxId{})
+
+	receipt, err := rt.Deploy(env, msg, ctx)
+	return rt, receipt, err
+}
+
+func Spawn(t *testing.T, rt *Runtime, path string) (*SpawnReceipt, error) {
+	msg := ReadFile(t, path)
+	gas := 1000000000
+	env := NewEnvelope(Address{}, Amount(10), TxNonce{Upper: 0, Lower: 0}, Gas(gas), GasFee(0))
+	ctx := NewContext(Layer(0), TxId{})
+
+	receipt, err := rt.Spawn(env, msg, ctx)
+	return receipt, err
+}
```

Comment:
Are these two supposed to be public, or is there a reason why they are?

Reply:
Actually not, I forgot about this rule in Golang
I'll change that